### PR TITLE
[ADD] partner event addon

### PR DESCRIPTION
**Pull request by antespi**
# Link partner and events

This module links partner with the events they are registered through a smart button.
It also includes a field for view and search only attended events.
And finally, partner column is visible on registration one2many list inside the event.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hi @Ehtaga could you review please this PR? Very similar to https://github.com/OCA/survey/pull/8
Thanks!

File: partner_event/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Link partner and events
```
Comment (pedrobaeza) [inline partner_event/README.rst:4]: Line on top

File: partner_event/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+=======================
+
+This module links partner with the events they are registered through a
```
Comment (pedrobaeza) [inline partner_event/README.rst:7]: s/partner/partners

File: partner_event/README.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+smart button.
+
+It also includes a field for view and search only attended events.
+
+And finally, partner column is visible on registration one2many list inside
+the event.
+
+
+Installation
```
Comment (pedrobaeza) [inline partner_event/README.rst:16]: Remove these empty sections

File: partner_event/README.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/199/8.0
+
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline partner_event/README.rst:36]: Remove this empty section

File: partner_event/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Link partner to events',
+    'version': '8.0.1.0.0',
+    'category': 'Marketing',
+    'author': 'Serv. Tecnolog. Avanzados - Pedro M. Baeza, '
```
Comment (pedrobaeza) [inline partner_event/__openerp__.py:30]: s/Tecnolog./Tecnol.

**Comment by pedrobaeza** [comment]
Minor nitpickings, but functionally :+1:

**Comment by Ehtaga** [comment]
Tested on runbot and code review: :+1:

**Comment by rafaelbn** [comment]
I don't get any results when searching by "Event attended registrations number". Please @JavierIniesta could you review too? Thanks

**Comment by rafaelbn** [comment]
Thanks! Tested in runbot and :+1:

**Pull request merged**